spec/metrics/relation_spec.rb

```diff
@@ -208,62 +227,57 @@
         end
       end
 
-      context "two args calculation methods" do
-        [
-          :percentile, :histogram, :top, :bottom
-        ].each do |method|
-          describe "##{method}" do
-            it do
-              expect(rel.where(user_id: 1).calc(method, :column_name, 10).to_sql)
-                .to eq "select #{method}(column_name,10) from \"dummy\" where (user_id=1)"
-            end
-          end
+      context "with aliases" do
+        it "select count as alias" do
+          expect(rel.count(:val, 'total').to_sql).to eq "select count(val) as total from \"dummy\""
+        end
+
+        it "select percentile as alias" do
+          expect(rel.percentile(:val, 90, 'p1').to_sql).to eq "select percentile(val,90) as p1 from \"dummy\""
         end
       end
     end
   end
 
   describe "#empty?" do
-    after(:each) { Influxer.reset }
-    it "should return false if has points" do
-      Influxer.client.stub(:query) { { points: [{ time: 1, id: 2 }] } }
+    it "return false if has points" do
+      allow(client).to receive(:query) { [{ "values" => [{ time: 1, id: 2 }] }] }
       expect(rel.empty?).to be_falsey
       expect(rel.present?).to be_truthy
     end
 
-    it "should return true if no points" do
-      Influxer.client.stub(:query) { {} }
+    it "return true if no points" do
+      allow(client).to receive(:query) { [] }
       expect(rel.empty?).to be_truthy
       expect(rel.present?).to be_falsey
     end
   end
 
   describe "#delete_all" do
-    it do
-      Timecop.freeze(Time.now) do
-        expect(rel.where(user_id: 1, dummy: 'q', timer: Time.now).delete_all)
-          .to eq "delete from \"dummy\" where (user_id=1) and (dummy='q') and (timer=#{Time.now.to_i}s)"
-      end
+    it "without tags" do
+      expect(rel.delete_all)
+        .to eq "drop series from \"dummy\""
     end
-  end
 
-  describe "#inspect" do
-    after(:each) do
-      Influxer.reset
+    it "with tags" do
+      expect(rel.where(dummy_id: 1, host: 'eu').delete_all)
+        .to eq "drop series from \"dummy\" where (dummy_id='1') and (host='eu')"
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver.